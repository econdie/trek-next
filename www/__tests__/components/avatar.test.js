import { shallow } from "enzyme";
import React from "react";
import renderer from "react-test-renderer";
import Avatar from "../../components/avatar";

describe("Avatar", () => {
  it("renders without crashing", () => {
    const av = shallow(<Avatar size="sm" />);
    expect(av.find("div").hasClass("avatar-sm")).toEqual(true);
  });

  it("Snapshot matches", () => {
    const component = renderer.create(<Avatar size="sm" />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
