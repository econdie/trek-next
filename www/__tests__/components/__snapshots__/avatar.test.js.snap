// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar Snapshot matches 1`] = `
<div
  className="avatar-sm"
  style={
    Object {
      "backgroundImage": "url(https://storage.googleapis.com/trek-next.appspot.com/public/avatar-default-sm.png)",
    }
  }
/>
`;
